DAOS-Support.md

```diff
@@ -1,7 +1,13 @@
 # DAOS Support
 
-[DAOS](https://github.com/daos-stack/daos) is supported as a backend storage system in dcp, dsync, and dcmp. The build instructions for
-enabling DAOS support can be found here:
+[DAOS](https://github.com/daos-stack/daos) is supported as a backend storage system in
+dcp, dsync, and dcmp. Custom serialization and deserialization for DAOS
+containers to and from a POSIX filesystem is provided with daos-serialize and
+daos-deserialize. These tools serialize a DAOS container and its associated
+metadata to an HDF5 file for storage on a POSIX filesystem. The container data
```

Comment:
Rewording a bit to be specific about naming which tool to use for generating the HDF file.
```suggestion
daos-deserialize. A DAOS container and its associated
metadata can be serialized to an HDF5 file for storage on a POSIX filesystem using daos-serialize. The container data
```